src/ui/components/VPNList.qml

```diff
@@ -11,7 +11,7 @@ ListView {
     id: list
 
     property var listName
-
+    height:contentHeight
```

Comment:
is this related? Does it change any other views?

Reply:
The list is just used in about-us. We need to limit the hight to it's content children otherwise the list grows to fill the parent.